tarorpc/taro.yaml

```diff
@@ -58,5 +58,9 @@ http:
       post: "/v1/taro/send"
       body: "*"
 
+    - selector: tarorpc.Taro.SubscribeSendAssetEventNtfns
```

Comment:
Not clear to me what happens when you ping this endpoint - do you get multiple events, or some handle to then stream events from, or something else?

Not blocking just curious.

Reply:
We use `lnd`'s WebSocket proxy that allows you to turn streaming RPCs into WebSockets: https://github.com/lightninglabs/taro/blob/main/server.go#L337

See https://github.com/lightningnetwork/lnd/blob/master/docs/rest/websockets.md for example code on how to use such endpoints.

Reply:
> See https://github.com/lightningnetwork/lnd/blob/master/docs/rest/websockets.md for example code on how to use such endpoints.

Nice docs, thanks for the link.